Dockerfile

```diff
@@ -0,0 +1,29 @@
+# This file is part of REANA.
+# Copyright (C) 2017 CERN.
+#
+# REANA is free software; you can redistribute it and/or modify it under the
+# terms of the GNU General Public License as published by the Free Software
+# Foundation; either version 2 of the License, or (at your option) any later
+# version.
+#
+# REANA is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR
+# A PARTICULAR PURPOSE. See the GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along with
+# REANA; if not, write to the Free Software Foundation, Inc., 59 Temple Place,
+# Suite 330, Boston, MA 02111-1307, USA.
+#
+# In applying this license, CERN does not waive the privileges and immunities
+# granted to it by virtue of its status as an Intergovernmental Organization or
+# submit itself to any jurisdiction.
+
+FROM python:3.5-alpine
```

Comment:
> "It might be good to standardize on some base everywhere in order to avoid surprises"

There has been discussions about moving to Alpine in order to achieve smaller image sizes. The usage of python:3.5-alpine in reana-server is intentional. Motivation for this is to find out possible problems with the way current Dockerfile(s) of various REANA components are written.

I've seen and read the blog post in the past.